### PR TITLE
feat: implement add asset

### PR DESCRIPTION
**Pull request by Alex-A4**
## Description

copilot:all

- Implement `AddAsset` from `InpageProvider`

## Type of Change

- [X] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: assets/translations/en.json [hunk 1]
```diff
@@ -347,5 +347,15 @@
...
+  "basicInteractionNotPermitted": "Basic interaction not permitted",
+  "permissionsNotGranted": "Permissions not granted",
+  "addAsset": "Add asset",
+  "versionWord": "Version",
+  "symbolWord": "Symbol",
+  "decimalsWord": "Decimals",
+  "ownerAddress": "Owner address",
+  "totalSupply": "Total supply",
+  "typeIsWrong": "Type is wrong"
+}
```
Comment (nesquikm) [inline assets/translations/en.json:361]: Newline at the eof.

File: assets/translations/ko.json [hunk 2]
```diff
@@ -347,5 +347,15 @@
...
+  "basicInteractionNotPermitted": "Basic interaction not permitted",
+  "permissionsNotGranted": "Permissions not granted",
+  "addAsset": "Add asset",
+  "versionWord": "Version",
+  "symbolWord": "Symbol",
+  "decimalsWord": "Decimals",
+  "ownerAddress": "Owner address",
+  "totalSupply": "Total supply",
+  "typeIsWrong": "Type is wrong"
+}
```
Comment (nesquikm) [inline assets/translations/ko.json:361]: Newline at the eof.

File: lib/data/models/asset_type.dart [hunk 3]
```diff
@@ -1,5 +1,11 @@
 import 'package:freezed_annotation/freezed_annotation.dart';
 
+/// This map needed because all input-related data stores in auto-generated
+/// webview models
+const assetTypeMap = {
+  'tip3_token': AssetType.tip3Token,
```
Comment (nesquikm) [inline lib/data/models/asset_type.dart:6]: We already have 'tip3_token' as string in the AssetType. Should we use one const string for the both cases?

**Review by nesquikm — CHANGES_REQUESTED** [review]

File: assets/translations/ko.json (hunk 2 above)
Comment (Alex-A4) [reply assets/translations/ko.json]: -

File: lib/data/models/asset_type.dart (hunk 3 above)
Comment (Alex-A4) [reply lib/data/models/asset_type.dart]: -

**Pull request merged**